langserve/server.py

```diff
@@ -866,6 +877,41 @@ async def playground(file_path: str, config_hash: str = "") -> Any:
             file_path,
         )
 
+    @app.post(namespace + "/feedback")
+    async def feedback(feedback_create_req: FeedbackCreateRequest) -> Feedback:
+        """
+        Send feedback on an individual run to langsmith
+        """
+
+        if not tracing_is_enabled():
+            raise HTTPException(
+                400,
+                "The feedback endpoint is only accessible when LangSmith is "
+                + "enabled on your LangServe server.",
+            )
+
+        feedback_from_langsmith = langsmith_client.create_feedback(
+            feedback_create_req.run_id,
+            feedback_create_req.key,
+            score=feedback_create_req.score,
+            value=feedback_create_req.value,
+            comment=feedback_create_req.comment,
+        )
```

Comment:
I don't currently have error handling here... we can add this as needed

Reply:
I think that's fine -- the request is already being validated, so likely any errors here aren't client side errors but server side errors, and by default they'll just show up as 500s which is good